x-pack/plugins/fleet/server/services/epm/packages/remove.ts

```diff
@@ -177,9 +181,27 @@ async function deleteKibanaAssets({
     const assetsToDelete = foundObjects.map(({ saved_object: { id, type } }) => ({ id, type }));
 
     await bulkDeleteSavedObjects(assetsToDelete, namespace, savedObjectsClient);
+    await deleteSloAssets(sloAssets, namespace, savedObjectsClient);
```

Comment:
What the behavior if the SLO do not exists? if it's not already handled I think it will be interesting to catch the error and log a warning but not fail the uninstallation